cmd/vic-machine/common/utils.go

```diff
@@ -46,7 +48,7 @@ func LogErrorIfAny(clic *cli.Context, err error) error {
 
 // CheckUnsupportedChars returns an error if string contains special characters
 func CheckUnsupportedChars(s string) error {
-	return checkUnsupportedChars(s, reUnsupp)
+	return checkUnsupportedChars(s, reUnsuppDatastore)
```

Comment:
I think these two methods CheckUnsupportedChars and CheckUnsupportedCharsDatastore are checking two different character set, why you need to change this method same to CheckUnsupportedCharsDatastore?